### ISSUE TITLE
A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_ScreenReader : The title and role of "Preview" dialog is not announced for screen reader user.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/azuredatastudio#11529]
**"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**
GitHubTags:#A11y_AzureDataStudioJuly2020;#A11y_July27_2020_TestPass;#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#MAC;#Screenreader;#VoiceOver;#A11ySev1;#Benchmark;#MAS1.3.1;#MAS4.1.2;#MAS2.4.2#FTP;

### Environment Details:
Application Name: Azure Data Studio
Application Version: 1.21.0-insider
Commit: eccf3cf
Date: 2020-07-24T09:28:31.172Z
VS Code: 1.48.0
Electron: 9.1.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina (Version 10.15.6 (19G73)
Screen Reader: VoiceOver
MAS References: MAS1.3.1, MAS4.1.2,MAS2.4.2

### Repro Steps:
1. Launch Azure Data Studio Insiders application.
2. If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3. Start screen reader, navigate to "Preview" control and activate it.
4. A dialog appears on the screen, listen if the name and role is announced to the screen reader users or not.

### Actual:
When screen reader users navigate to the "Preview" control and activate it, a dialog appears on the screen but the screen reader user are not aware of the same as the name and role of the dialog is not announced to inform them.
 
### Expected:
As soon as the dialog appears on the screen title of the dialog should b announced as the name and the role as 'dialog' should be announced for the screen reader users.

### User Impact:
If proper information is not conveyed to the screen reader users for the changes appearing on the screen then they will miss out on the information that is available for the other users.

### Attachment link for Reference:
[11529_A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_ScreenReader The title and role of "Preview" dialog is not announced for screen reader user.zip](https://github.com/microsoft/azuredatastudio/files/4985256/11529_A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_ScreenReader.The.title.and.role.of.Preview.dialog.is.not.announced.for.screen.reader.user.zip)

**Comment by alanrenmsft** [comment]
I have simplified the preview information to use tooltip to show the text provided by the old dialog.

**Comment by Raghu-addepalli** [comment]
Issue is fixed Hence closing the bug and added attachment.
Githubtags:  #Closed;
[Uploading 11529_Fixed.zip…]()
